packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -139,6 +141,7 @@ export function finishUploading(state, { payload: { id, resource } }) {
         ? {
             ...item,
             resource,
+            previousResourceId: item.resource.id,
```

Comment:
What is the difference between `previousResourceId` and `originalResourceId`?

Reply:
`originalResourceId`:

You have an existing video ("A") on the page and you click "mute". A new video ("B") will be added to the queue for muting.  It references "A" as the `originalResourceId`. Maybe `parentResourceId` is a better term, not sure.

`previousRecordId`:

A new item is added to the queue. At first it's a local file with blob URL and a random ID.
After uploading to WordPress, the random ID gets replaced by the WP attachment ID. The previous one is stored as `previousRecordId`, so that we can update all instances of the local file with the final resource + attachment ID.

Reply:
So it is a localResourceId then? Maybe we could rename it to make clear what it is?

Reply:
It's the ID of the previous instance of this resource, which happened to be a local resource or placeholder.

`localResourceId` doesn't hit the nail on the head IMO.

Given that this is only used in few places, inline documentation should suffice for this.